vignettes/queries_dataset.Rmd

```diff
@@ -0,0 +1,178 @@
+---
+title: "Queries dataset documentation"
+output:
+  rmarkdown::html_vignette:
+  toc: false
+vignette: >
+  %\VignetteIndexEntry{Queries dataset documentation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+  ```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+To support the analysis of the safety analysis, it is quite common to define specific grouping of events. One of the most common way is to group events or medications by a specific medical concept such as a Standard MedDRA Queries or WHO-Drug Standardized Drug Grouping . 
+
+
+To help with the derivation of these variables, the admiral function `derive_vars_query()` can be used.
+This function takes as input the dataset (`dataset`) where the grouping must occurs (e.g `ADAE`) and
```

Comment:
Replace with either `where the grouping must occur` or `where the grouping occurs`